### ISSUE TITLE
Android compatibility

### ISSUE THREAD

**Issue by mjibril** [LeMaker/fex_configuration#2]
Will the 7 inch Lemaker LCD screen work with Android on bananapi pro?

**Comment by Tony-HIT** [comment]
Yes. The 7inch lcd work for Android. But we need customize the Android image.
